src/test/addrman_tests.cpp

```diff
@@ -15,9 +15,13 @@ class CAddrManTest : public CAddrMan
     uint64_t state;
 
 public:
-    CAddrManTest()
+    CAddrManTest(bool makeDeterministic=true)
```

Comment:
Well there is no need for constructor argument, just call `MakeDeterministic()`.